### PR TITLE
New: Custom formats in episode history

### PR DESCRIPTION
**Pull request by mynameisbogdan**
#### Database Migration
NO

#### Description
image

### CODE REVIEW

**Comment by stevietv** [comment]
this will certainly help when providing support via screenshots to identify CF matches

File: frontend/src/Episode/History/EpisodeHistoryRow.js [hunk 1]
```diff
@@ -102,14 +101,28 @@ class EpisodeHistoryRow extends Component {
...
         <RelativeDateCellConnector
           date={date}
         />
 
-        <TableRowCell className={styles.details}>
+        <TableRowCell className={styles.actions}>
           <Popover
             anchor={
-              <Icon
+              <IconButton
```
Comment (markus101) [inline frontend/src/Episode/History/EpisodeHistoryRow.js:125]: Why is it changing to a button?

File: frontend/src/Episode/History/EpisodeHistoryRow.js [hunk 2]
```diff
@@ -102,14 +101,28 @@ class EpisodeHistoryRow extends Component {
           />
         </TableRowCell>
 
+        <TableRowCell>
+          <EpisodeFormats formats={customFormats} />
+        </TableRowCell>
+
+        <TableRowCell>
+          <Tooltip
```
Comment (markus101) [inline frontend/src/Episode/History/EpisodeHistoryRow.js:109]: Doesn't seem super useful to have a tooltip with the custom formats when they're always shown in the column to the left.

File: frontend/src/Episode/History/EpisodeHistoryRow.js (hunk 1 above)
Comment (markus101) [reply frontend/src/Episode/History/EpisodeHistoryRow.js:125]: I understand that grouping these saves horizontal space, but information under `Actions` doesn't seem right and we should consider removing the column title.

File: frontend/src/Episode/History/EpisodeHistoryRow.js (hunk 2 above)
Comment (stevietv) [reply frontend/src/Episode/History/EpisodeHistoryRow.js:109]: would make sense to me to remove the tooltip indeed!

**Review by markus101 — APPROVED** [review]

**Pull request merged**